### ISSUE TITLE
oclint failed with error code 6

### ISSUE THREAD

**Issue by ItghostFan** [oclint/oclint#622]
### Issue Summary
We use oclint-json-compilation-database failed with error code 6.
Actual error code 6 generated by  oclint.

### Environment
- Operation system name and version:
  macOS Big Sur
  11.3.1 (20E241)
- OCLint version:
  OCLint (http://oclint.org/):
  OCLint version 20.11.
  Built Nov 14 2020 (10:19:05).
- How OCLint is installed: local build? prebuilt binary downloaded from github? homebrew install? others?
  brew install

### Reproduction Steps
We use [run-sonar-swift.sh](https://raw.githubusercontent.com/Backelite/sonar-swift/master/sonar-swift-plugin/src/main/shell/run-sonar-swift.sh).

#### Sample code

```shell
oclint-json-compilation-database -v -e Vendor -e Pods -- -rc LONG_LINE=250 -max-priority-1 10000 -max-priority-2 10000 -max-priority-3 10000 -report-type pmd -o sonar-reports/oclint.xml
```

ERROR - Command 'oclint-json-compilation-database -v -e Vendor -e Pods -- -rc LONG_LINE=250 -max-priority-1 10000 -max-priority-2 10000 -max-priority-3 10000 -report-type pmd -o sonar-reports/oclint.xml' failed with error code: 6

### Wants
What's error code 6 meaning? How to fix?

**Comment by ryuichis** [comment]
Error 6 means compilation error. By looking at your description, I wonder if you try to use OCLint on Swift files. In addition, if you are using sonarcube, you probably need to talk to their customer service.
